modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserAccountResourceTest.java

```diff
@@ -1837,6 +1840,67 @@ private void _testGetUserAccountsPageWithCustomFields() throws Exception {
 			userAccount);
 	}
 
+	private void _testGetUserAccountWithMoreExternalReferenceCodes()
+		throws Exception {
+
+		User user = UserTestUtil.addUser();
+
+		_userLocalService.addRoleUser(_role.getRoleId(), user.getUserId());
+
+		_userLocalService.addOrganizationUser(
+			_organization.getOrganizationId(), user.getUserId());
+
+		_userLocalService.addUserGroupUser(
+			_userGroup.getUserGroupId(), user.getUserId());
+
+		_userLocalService.addGroupUser(
+			testGroup.getGroupId(), user.getUserId());
+
+		UserAccount userAccount = userAccountResource.getUserAccount(
+			user.getUserId());
+
+		OrganizationBrief[] organizationBriefs =
+			userAccount.getOrganizationBriefs();
+
+		Assert.assertTrue(organizationBriefs.length > 0);
+
+		OrganizationBrief organizationBrief = organizationBriefs[0];
+
+		Assert.assertEquals(
+			_organization.getExternalReferenceCode(),
+			organizationBrief.getExternalReferenceCode());
+
+		RoleBrief[] roleBriefs = userAccount.getRoleBriefs();
+
+		Assert.assertTrue(roleBriefs.length > 1);
```

Comment:
I noticed that you check that the array has 1+ length compared to everywhere else where you check for > 0. Could be intentional and also correct but wanted to point it out to make sure it's not a small mistake.

Reply:
It was a mistake on my part, it should be > 0

Reply:
This is fine, because we need to know that there are two elements to get the [1]